myhpi/feed/views.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+import json
+
+from django.core.exceptions import MultipleObjectsReturned, SuspiciousOperation
+from django.http import HttpResponse
+from django.views import View
+
+from myhpi.feed.models import NewsFeed, Post, PostAccount
+
+
+class PostFeedEntryView(View):
```

Comment:
is there a reason why you are not using the [wagtail API](https://docs.wagtail.org/en/stable/advanced_topics/api/v2/configuration.html) for this?

Reply:
Did not find them when googling (searched for creating a django API). Will look into it.

Reply:
Wagtail API is read only, so does not fit the use case.